app/javascript/packs/application.tsx

```diff
@@ -490,5 +491,17 @@ document.addEventListener('turbolinks:load', () => {
   highlightAll()
 })
 
+document.addEventListener('turbolinks:visit', () => {
```

Comment:
@iHiD This is how I plan to do this, but I might just trigger this to turn on only on actions that redirect, instead of  the overlay appearing every time it is loading between pages.